### DISCUSSION TITLE
Virtual Controller Remapper's bug

### DISCUSSION THREAD

**Discussion by xcloudbrazie** [redphx/better-xcloud#536] [General]
I am experiencing issues with Better Xcloud. I am unable to edit the mouse and keyboard controls, and the configured keys do not appear. I have tested it on two different computers, and the problem persists. I also tried playing on both Chrome and Edge, but the issue continues.

![Sem título-1](https://github.com/user-attachments/assets/de0605e3-1c2e-4b56-bad4-869ab67f984c)
![Sem título-2](https://github.com/user-attachments/assets/00005232-02f0-49b0-bde4-d351fb5fa9a9)

**Comment by redphx** [comment]
Test this: https://github.com/redphx/better-xcloud/raw/refs/heads/typescript/dist/better-xcloud.user.js

**Reply by xcloudbrazie** [reply]
![image](https://github.com/user-attachments/assets/83380422-de2d-4c49-b218-56419e9090cd)
The problem persists...

**Reply by redphx** [reply]
Fixed. Try install the same link again.

**Reply by xcloudbrazie** [reply]
![image](https://github.com/user-attachments/assets/c6003ac1-309a-49bb-9dc1-13b80fffe343)
The error continues.. I don't know what else to do anymore..

**Reply by redphx** [reply]
Try again in 10 mins
or use this link: https://github.com/redphx/better-xcloud/raw/5b67b4c37dc8186f9a678f02578e616ed57d576b/dist/better-xcloud.user.js

**Reply by xcloudbrazie** [reply]
Actually, the buttons and mouse work in the game, but I can't edit or see which buttons are configured. If I could at least see the default buttons.

**Reply by redphx** [reply]
This should be the final fix: https://github.com/redphx/better-xcloud/raw/53442557e152f202fbfa6bcccadde8878649c04d/dist/better-xcloud.user.js

**Reply by xcloudbrazie** [reply]
![image](https://github.com/user-attachments/assets/c3df07c4-ea3f-4077-92e1-62d92995a667)
It worked... thank you so much!! You’re amazing!!

**Reply by redphx** [reply]
thanks for testing
